Chocolatey/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,15 @@
+﻿$ErrorActionPreference = 'Stop';
+
+$packageName  = 'FlowLauncher'
+$toolsDir     = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+
+$packageArgs = @{
+  packageName   = $packageName
+  fileType      = 'exe'
+  url           = 'https://github.com/Flow-Launcher/Flow.Launcher/releases/download/v1.0.0/Flow-Launcher-v1.0.0.exe'
```

Comment:
Do we want to change the file name so the url is https://github.com/Flow-Launcher/Flow.Launcher/releases/latest/download/Flow-Launcher.exe ?

If so lets do this in a seperate PR